.pre-commit-config.yaml

```diff
@@ -41,7 +41,7 @@ repos:
     hooks:
       - id: mypy
         exclude: docs
-        additional_dependencies: [xarray, types-pkg_resources]
+        additional_dependencies: [xarray, types-pkg_resources, numpy<1.21]
```

Comment:
@malmans2 `xarray` types incompatible with `numpy` released yesterday. I think it has been fixed upstream in `xarray`, so remember to remove also because the ci bot environment is OK (I guess due to caching or pre-built env).